task_runner

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+TARGET=skbio
+
+if [ ${2} ]; then
+  TARGET=${2}
+fi
+
+COMMAND=${1}
+
+case "${COMMAND}" in 
+  "tests" )
+    set -x
+    nosetests ${TARGET} --with-doctest --ignore DO_NOT_IGNORE_ANYTHING
```

Comment:
I think we'll need a way to turn on/off `--with-doctest` and `--with-coverage` (assuming that Travis will be using this). doctests are only run with Python 2.x. Maybe default to running both of these?